ZEP/instructions/zep0000.md

```diff
@@ -0,0 +1,216 @@
+
+Because the ZEPs are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal.
+
+## ZEP audience
+
+The typical primary audience for ZEPs is the developers working on Zarr and its various implementations, the Zarr steering council as well as the Zarr community.
+
+The broader Zarr community may also choose to use the process to document expected feature additions and to manage complex design coordination problems that
+require collaboration across multiple projects.
+
+## ZEP types
+
+> Specification ZEP
+
+Specification ZEPs deal with the changes related to [Zarr storage specification](https://zarr.readthedocs.io/en/stable/spec.html). The SPEC ZEP (abbreviation of Specification ZEP) introduces changes in the core specification of Zarr. The changes related to core specification follow a strict and thorough review process and should be adopted by everyone in the Zarr community.
+
+- Core protocol ZEP
+
+Describes a ZEP which involves changes in the core specification of Zarr. Core protocol ZEPs (commonly known as Core ZEPs) are a part of SPEC ZEPs and apply to [Zarr storage specification](https://github.com/zarr-developers/zarr-specs) and its various implementations under the [Zarr Implementations](https://github.com/zarr-developers/zarr_implementations) GitHub repo. The core protocol ZEPs should be adopted by every implementation of Zarr and, in general, the overall Zarr community. Core ZEPs must go through a thorough review process, including involvement, discussion, and voting from the author of the proposed ZEP, Zarr Steering Council, author(s) of various Zarr implementations, and open-source projects/research groups using Zarr and the general Zarr community. The general advice is that everyone should be made aware of the change by introducing core ZEP. Core protocol ZEP require community consensus, and users are typically not free to ignore them.
+
+Note: Currently, Specification ZEPs only deal with the core storage specification of Zarr. The scope of SPEC ZEPs will be extended in the future.
+
+> Informational ZEP
+
+Describes a ZEP design issue or provides general guidelines or information to the Zarr community but does not propose a new feature. Informational ZEPs do not necessarily represent Zarr community consensus or recommendation, so users and implementers are free to ignore informational ZEPS or follow their advice.
+
+> Process ZEP
+
+Describes a new process around Zarr and its implementations or proposes a change to(or an event in) a process. They may propose an implementation, but not to Zarr’s codebase; they require community consensus; unlike informational ZEPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Zarr’s specification development. Any meta-ZEP is also considered a Process ZEP.
```

Comment:
> They may propose an implementation, but not to Zarr’s codebase;

Not sure what that means, could you give an example here @MSanKeys963? The rest of the explanation of the different ZEPs seem very clear to me :+1:

Reply:
I guess my confusion resulted from associating implementations with code-implementations, thanks for the clarification @MSanKeys963